### ISSUE TITLE
[Question] How to write some docs for input types

### ISSUE THREAD

**Issue by massimilianobraglia** [thecodingmachine/graphqlite#337]
Hey there! Is it possible to write some docs for my input types?

When I use the `@Field` annotation, I can use the phpdoc and the comment is transferred to the GQL field. When using a factory this won't happen 😞 

Examples:

This is not working:
```php
class MyFactory
{
    /**
     * @param string $value This comment won't be visible anywhere
     * @GQL\Factory(name="MyInput", default=true)
     */
    public function createMyInput(string $value)
    {
        return new MyObject($value);
    }
}

class MyObject
{
    /**
     * Even this comment won't be visible anywhere
     */
    public string $value;

    public function __construct(string $value)
    {
        $this->value = $value;
    }
}
```

This is working

```php
/**
 * @GQL\Type()
 */
class MyType
{
    /**
     * This comment is visible as docs for this field.
     * @GQL\Field()
     */
    public function getValue(): string
    {
        return 'value';
    }
}

**Comment by oojacoboo** [comment]
Please see #289

**Comment by massimilianobraglia** [comment]
Hey @oojacoboo, thanks for your answer! Any thoughts about the release of that feature?

**Comment by oojacoboo** [comment]
I believe this will be released in the upcoming 4.2 update.  I'm still waiting for clarity around this though.  We should have this update tagged and released soon.
